llarp/exit/endpoint.cpp

```diff
@@ -98,7 +98,7 @@ namespace llarp
       auto lastPing = path->LastRemoteActivityAt();
       if(lastPing == 0 || (now > lastPing && now - lastPing > timeout))
         return now > m_LastActive && now - m_LastActive > timeout;
-      else if(lastPing)
+      if(lastPing)
```

Comment:
this part here is needed because of the other return path

Reply:
which other return path? they’re functionally equivalent afaict.

Reply:
Corrected either way.